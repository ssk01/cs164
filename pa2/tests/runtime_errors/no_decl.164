print 1
print x
print 2
