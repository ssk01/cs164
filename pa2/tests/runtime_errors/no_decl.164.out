1
Error: x is not declared
